.github/workflows/ci.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8 pytest
+        pip install flake8 pytest pytest-cov pytest-timeout
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```

Comment:
It should work if we install as described in the readme:

    pip install pipenv
    pipenv install --dev